src/hooks/useAxios.js

```diff
@@ -9,17 +9,24 @@ import axios from "axios";
  *
  */
 
-export const useAxios = (url) => {
+export const useAxios = (url, encodedToken = "") => {
```

Comment:
suggestion - here you can have `type` as an argument too so that it can be used for `post` and `delete` requests too.